src/tutorials/first_game_br.md

```diff
@@ -0,0 +1,166 @@
+
+<!--- TODO add image of decal properties --->
+
+### Update the Template
+
+Once all changes are completed, right click the **Helper_Plane** and select **Update Template From This**. After, delete **Helper_Plane** from the **Hierarchy**.
+
+<!--- TODO add image of updating template --->
+
+### Test the Game
+
+The airplane template should now be modified in game and is ready to test. The game requires at least two players to begin, so make sure to enable **Multiplayer Preview Mode** before pressing **Play**.
+
+![TwoPlayers](../img/MyFirstMultiplayer/TwoPlayers.png){: .center loading="lazy" }
+
+## Edit the Drop Zone
+
+The "Drop Zone" is a cylindrical area that represents where the player can drop from the plane. The plane will spawn near the edge of the cylinder at random and then fly across to the other side.
+
+<!--- TODO image of drop zone cylinder --->
+
+### Focus on the Drop Zone
+
+Inside the **Hierarchy**, navigate to the **Gameplay Settings > State Settings > Round Start Skydive > ServerContext** folder. Select the **DropArea** and press ++F++ to focus the scene on the large cylinder mesh.
+
+<!--- TODO image of hierarchy --->
+
+### Scale the Drop Zone
+
+Press ++R++ to enter scale mode. Change the X and Y scale so the Drop Zone is a different size.
+
+<!--- TODO image of larger drop zone --->
+
+## Change the Falling Player Movement
+
+The player has two different falling phases. When the player drops from the plane, it enters a free fall known as the **Skydive** phase. When the player presses ++space++ while skydiving then it will transition to **Parachute** phase.
+
+### Change Skydive Settings
+
+Navigate to **Gameplay Settings > State Settings > Round Start Skydive > ServerContext**. Select the **SkydivePlayerSettings** and open the Properties window. Increase the **Gravity Scale** property to `3.2` and notice how the player will fall down twice as fast with no parachute.
+
+<!--- TODO image of properties window --->
+
+### Change Parachute Settings
+
+In the same **ServerContext** folder as the **SkydivePlayerSettings**, select the **ParachutePlayerSettings**. In the **Properties** window, change the **Ground > Max Speed** property to `2400`. This will increase the max speed moving forward while parachuting.
+
+<!--- TODO image of properties window --->
+
+## Changing the Play Zone
+
+The "Play Zone" is a cylindrical area where the player will *not* receive constant damage. The Play Zone will shrink in size for a certain amount of intervals.
+
+### Change the Play Zone Settings
+
+Navigate to **Gameplay Settings > Zone Settings > Constricting Play Zone** and open the **Properties** window. The object has many **Custom Properties** that will be described below:
+
+| Property Name | Description |
+| --- | --- |
+| Phase Count | The number of phases. The Play Area will shrink in size as the phases increase. It will stop shrinking once the last phase occurs. |
```

Comment:
So I really like the idea that you're using "number" for integers and "amount" for floats. I think we should keep that convention, so double-check if any of these are, in fact, integers, and swap to "number" for those.

Reply:
Ooh good point, I did that subconsciuosly but I just doublechecked and they do in fact line up.